packages/fieldset/test/lion-fieldset.test.js

```diff
@@ -614,24 +614,32 @@ describe('<lion-fieldset>', () => {
       });
     });
 
-    it('0 is a valid value to be serialized', async () => {
+    it('serializes 0 and empty string, they are valid values', async () => {
+      // implementation should just use `!= null` comparator to check for undefined and null.
       const fieldset = await fixture(html`
       <${tag}>
         <${childTag} name="price"></${childTag}>
+        <${childTag} name="firstName"></${childTag}>
       </${tag}>`);
       await nextFrame();
       fieldset.formElements.price.modelValue = 0;
-      expect(fieldset.serializeGroup()).to.deep.equal({ price: 0 });
+      fieldset.formElements.firstName.modelValue = '';
+      expect(fieldset.serializedValue).to.deep.equal({ price: 0, firstName: '' });
     });
 
-    it('__serializeElements serializes 0 as a valid value', async () => {
-      const fieldset = await fixture(html`<${tag}></${tag}>`);
+    it('does not serialize null and undefined, they are invalid values', async () => {
+      const fieldset = await fixture(html`
+      <${tag}>
+        <${childTag} name="price"></${childTag}>
+        <${childTag} name="firstName"></${childTag}>
+      </${tag}>`);
       await nextFrame();
-      const elements = [{ serializedValue: 0 }];
-      expect(fieldset.__serializeElements(elements)).to.deep.equal([0]);
+      fieldset.formElements.price.modelValue = null;
+      fieldset.formElements.firstName.modelValue = undefined;
```

Comment:
Re-evaluate undefined, see what the platform does, might make sense to include this.

Reply:
See also separate Unparseable story